modules/ROOT/pages/branded_ios_app/publishing_ios_app_10.adoc

```diff
@@ -0,0 +1,58 @@
+= Additional Server Configuration 
+:apple-universal-link-support-url: https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html
+:admin_manual: https://doc.owncloud.com/server/administration_manual/index.html
+:apple-app-site-association-file-url: https://developer.apple.com/documentation/security/password_autofill/setting_up_an_app_s_associated_domains
+
+== Add Support for Apple Universal Links
+
+There's some special changes that need to be made. 
+Quoting from Apple's official documentation on {apple-universal-link-support-url}[Universal Link Support]:
+
+[quote]
+____
+Adding support for universal links is easy. 
+There are three steps you need to take:
+
+. Create {apple-app-site-association-file-url}[an apple-app-site-association file] that contains JSON data about the URLs that your app can handle.
+. Upload the apple-app-site-association file to your HTTPS web server. You can place the file at the root of your server or in the `.well-known` subdirectory.
+. Prepare your app to handle universal links. 
+____
+
+The `apple-app-site-association` will be generated by xref:branded_ios_app/publishing_ios_app_7.adoc#generate-ios-app[ownBrander], and needs to be served statically on your site through HTTPS. 
+You can safely place it on the root folder of your ownCloud installation (e.g., `/var/www/owncloud`). 
+
+=== Apache Configuration
+
+To achieve the second requirement, some changes will also need to be completed on your Apache configuration.
+
+If you configured your installation with the official {admin_manual}[Admin Manual], your Apache `owncloud.conf` file should include the following:
+
+[source,apacheconf]
+....
+# Create two alias for the file (for compatibility reasons):
+AliasMatch "^/(\.well-known/)?apple-app-site-association$" "/var/www/owncloud/apple-app-site-association"
```

Comment:
This is only one alias, the second one is missing.
I guess that the missing alias is:
`AliasMatch "/owncloud/apple-app-site-association" "/var/www/owncloud/apple-app-site-association"`
Then both the `.well-known/apple-app-site-association` and `apple-app-site-association` subdirs are defined.

pls see the definition and difference of AliasMatch for more details
https://httpd.apache.org/docs/2.4/mod/mod_alias.html#aliasmatch

Reply:
Can you review the revised text to ensure that I've understood you correctly, pls?